src/pymola/parser.py

```diff
@@ -23,7 +23,7 @@
 #  - Named function arguments (note that either all have to be named, or none)
 #  - Make sure slice indices (eventually) evaluate to integers
 
-class ModelicaFile:
+class ModelicaFile(object):
```

Comment:
We don't really care about new/old style classes (a Python 2 thing) anymore do we?